examples/refresh.mjs

```diff
@@ -26,20 +26,18 @@ const settings = await load(connectionString, {
     refreshOptions: {
         enabled: true,
         watchedSettings: [{ key: "app.settings.sentinel" }],
-        refreshIntervalInMs: 10 * 1000 // Default value is 30 seconds, shorted for this sample
+        refreshIntervalInMs: 10_000 // Default value is 30 seconds, shorted for this sample
     }
 });
 
 console.log("Using Azure portal or CLI, update the `app.settings.message` value, and then update the `app.settings.sentinel` value in your App Configuration store.")
 
 // eslint-disable-next-line no-constant-condition
 while (true) {
-    // Refreshing the configuration setting
-    await settings.refresh();

```

Comment:
This is a console app. It makes sense to keep the `await`. It will be helpful if we can update the comments to indicate this is a blocking call and users can remove `await` if they want it to be asynchronous. What matters is that users understand the implications and make conscious choices. There is nothing wrong with having `await`.